<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Employee Management System</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="3">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Open_File">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QPushButton" name="opendb_btn">
          <property name="text">
           <string>Open existing Employee DB</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="newdb_btn">
          <property name="text">
           <string>New Employee DB</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Login_Page">
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="3" colspan="3">
         <widget class="QLineEdit" name="name_text">
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="placeholderText">
           <string>Enter your Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4" colspan="2">
         <widget class="QRadioButton" name="Emp">
          <property name="text">
           <string>Employee</string>
          </property>
         </widget>
        </item>
        <item row="7" column="4" colspan="2">
         <widget class="QPushButton" name="login_btn">
          <property name="toolTip">
           <string>Login</string>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3">
         <widget class="QRadioButton" name="Admin">
          <property name="text">
           <string>Admin</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3" colspan="2">
         <widget class="QLineEdit" name="pass_text">
          <property name="echoMode">
           <enum>QLineEdit::PasswordEchoOnEdit</enum>
          </property>
          <property name="placeholderText">
           <string>Enter Password</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="name_lbl">
          <property name="text">
           <string>Name: </string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="pass_lbl">
          <property name="text">
           <string>Password: </string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="ut_lbl">
          <property name="text">
           <string>User Type: </string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QPushButton" name="register_btn">
          <property name="toolTip">
           <string>Register</string>
          </property>
          <property name="text">
           <string>Register</string>
          </property>
         </widget>
        </item>
        <item row="5" column="5">
         <widget class="QPushButton" name="rand_pass_btn">
          <property name="toolTip">
           <string>Generate a random password.</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icons/random_pass.png</normaloff>../Icons/random_pass.png</iconset>
          </property>
         </widget>
        </item>
        <item row="6" column="1" colspan="6">
         <widget class="QLabel" name="error_msg">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Admin_Main_Page">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" colspan="3">
         <widget class="QLineEdit" name="search_txt">
          <property name="placeholderText">
           <string>Search for an emplyee by his name.</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="search_btn">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Icons/search.png</normaloff>../Icons/search.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QFrame" name="Sidebar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="0">
            <widget class="QLabel" name="l_lbl">
             <property name="text">
              <string>Lower Limit</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="l_limit"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="des_lbl">
             <property name="text">
              <string>Designations: </string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="u_lbl">
             <property name="text">
              <string>Upper Limit</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="u_limit"/>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="salary_lbl">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Salary:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="des_data">
             <property name="editable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1" colspan="3">
         <widget class="QTableWidget" name="Emp_View">
          <property name="columnCount">
           <number>8</number>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Gender</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Salary</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Date of Joining</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Designation</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Age</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Experience</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Department</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Employee_EMP_Page"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
